### DISCUSSION TITLE
Unsure why supabase is not returning values from foreign table

### DISCUSSION THREAD

**Discussion by Electric-Dragon** [supabase/supabase#5807] [Questions] [answered]
I have a query that also selects from a foreign table. The current table already has a foreign key relationship. Here is my query
```
const { data, error } = await supabase
        .from('comments')
        .select('content,created_at,user(pfp,username)')
        .eq('chapter', chapterid)
        .limit(7)
        .select()
```
This is what is returned:
`{id: 5, created_at: "2022-03-03T18:16:20.592125+00:00", user: "7a2750e7-0d25-4a40-9236-ce7a42d61bd2", chapter: "1b4e4595-089e-457c-ac89-9ed4eace0ca9", content: "Woahh"}`

My comments table
[![Comments Table][1]][1]

The table the user column references (profile table)

[![Profile table][2]][2]

  [1]: https://i.stack.imgur.com/4QU58.png
  [2]: https://i.stack.imgur.com/DMrKe.png

**Comment by GaryAustin1** [comment, marked as answer]
You have an extra .select() at end which fetches all columns...

**Reply by Electric-Dragon** [reply]
Wow thanks a lot!
